### PR TITLE
PD-09 - Update billing docs

### PR DESCRIPTION
**Pull request by smithjilks**
Resolves [#9](https://github.com/ultravioletrs/prism-docs/issues/9)

### CODE REVIEW

File: docs/billing.md [hunk 1]
```diff
@@ -13,38 +13,31 @@ Currently, billing plans are only based on the maximum number of computations th
 Only an admin can create a billing plan which will be viewed by the organizations permitted to view it. In order to create a billing plan, the following steps need to be followed:
 
 ```bash
-curl -sSiX POST https://prism.ultraviolet.rs/createplan -H "Content-Type: application/json" -H "Authorization: Bearer <user_token>" -d @- << EOF
+curl -sSiX POST https://prism.ultraviolet.rs/billing/createplan -H "Content-Type: application/json" -H "Authorization: Bearer <user_token>" -d @- << EOF
```
Comment (SammyOina) [inline docs/billing.md:16]: the docs are cutomer facing, only admin users (ultraviolet) should be able to do this, so this doc should be on the private repo

File: docs/billing.md [hunk 2]
```diff
@@ -53,137 +46,219 @@ Response:
...
+   ...
+  ]
+}
+```
+On the UI the steps are as follows:
+
+1. Click on 'Admin' on the side navigation to view the plans.
+   ![Admin Page](img/ui/plans%20list.png)
+
 ### Delete Plan
```
Comment (SammyOina) [inline docs/billing.md:151]: same goes for deleting plans

Comment (SammyOina) [inline docs/billing.md:151]: same goes for deleting plans

Comment (SammyOina) [inline docs/billing.md:151]: same goes for deleting plans

File: docs/billing.md [hunk 3]
```diff
@@ -53,137 +46,219 @@ Response:
...
+Date: Tue, 04 Jun 2024 10:12:04 GMT
 ```
 
+On the UI the steps are as follows:
+
+1. Click on 'Admin' on the side navigation.
+2. Click on the delete button on any of the plans you'd like to delete.
+   ![Admin Page](img/ui/plans%20list.png)
+
 ### Update Plan
```
Comment (SammyOina) [inline docs/billing.md:182]: same here, customers should only be able to list, view and subscribe to a plan

Comment (SammyOina) [reply docs/billing.md:182]: and set their customer details and update them

Comment (SammyOina) [reply docs/billing.md:182]: and set their customer details and update them

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: docs/billing.md [hunk 4]
```diff
@@ -6,184 +6,81 @@ The billing service allows the organization to pay for the services used in the
 
 Currently, billing plans are only based on the maximum number of computations that an organization or user can have in the system. When an admin creates a plan, they limit the amount of computations that an organization can have based on the amount paid by the user. This is still a work in progress and the actual metrics that should be limited will be changed based on internal discussions.
```
Comment (SammyOina) [inline docs/billing.md:7]: please also update this to be customer facing

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]
remove curl and leave only ui
also remove all admin parts and leave only customer facing parts

File: docs/img/ui/update customer dialog.png
Comment (SammyOina) [inline docs/img/ui/update customer dialog.png:1]: outdated

File: docs/img/ui/plans list.png
Comment (SammyOina) [inline docs/img/ui/plans list.png:1]: remove admin related items from docs

File: docs/img/ui/edit plan page.png
Comment (SammyOina) [inline docs/img/ui/edit plan page.png:1]: same here

File: docs/img/ui/create plan dialog.png
Comment (SammyOina) [inline docs/img/ui/create plan dialog.png:1]: same

File: docs/img/ui/create customer dialog.png
Comment (SammyOina) [inline docs/img/ui/create customer dialog.png:1]: also out of date

Comment (SammyOina) [inline docs/img/ui/create customer dialog.png:1]: also out of date

File: docs/img/ui/admin page.png
Comment (SammyOina) [inline docs/img/ui/admin page.png:1]: remove

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

File: docs/img/ui/checkout page.png
Comment (SammyOina) [inline docs/img/ui/checkout page.png:1]: switch this image

File: docs/img/ui/billing plans.png
Comment (SammyOina) [inline docs/img/ui/billing plans.png:1]: same here

File: docs/img/ui/billing customer.png
Comment (SammyOina) [inline docs/img/ui/billing customer.png:1]: same here

**Review by SammyOina — CHANGES_REQUESTED** [review]
update the images some of these still mention organizations rather than workspaces

**Review by SammyOina — APPROVED** [review]

**Comment by dborovcanin** [comment]
@smithjilks Please resolve conflicts.

**Pull request merged**